app/controllers/idv/in_person/usps_locations_controller.rb

```diff
@@ -64,6 +64,25 @@ def add_proofing_component
           update(document_check: Idp::Constants::Vendors::USPS)
       end
 
+      def localized_locations(locations)
+        return nil if locations.nil?
+        locations.map do |location|
+          location.slice(
+            :address,
+            :city,
+            :distance,
+            :name,
+            :zip_code_4,
+            :zip_code_5,
+            :is_pilot,
+          ).merge(
+            saturday_hours: EnrollmentHelper.localized_hours(location[:saturday_hours]),
+            sunday_hours: EnrollmentHelper.localized_hours(location[:sunday_hours]),
+            weekday_hours: EnrollmentHelper.localized_hours(location[:weekday_hours]),
+          )
+        end
```

Comment:
just realized this whole thing is repeated in two controllers, what if we made `EnrollmentHelper.localized_location` or some other helper that could be reused?

Reply:
Yeah, I thought about that and I felt I was already deep enough in the rabbit hole to open a PR. I'm on board with it.

Reply:
Forgot that slice/merge doesn't work because location is a `PostOffice` rather than a hash. I've moved it to the EnrollmentHelper and switched to calling the attributes as methods rather than `#[]` to indicate that.

Reply:
yay big fan of methods

Reply:
oooo or what if we had `PostOffice#localized` which returns another instance of `PostOffice` (maybe with a `localized?: true` snuck in there to make sure we can't double-localize

Reply:
I experimented with something along those lines, but it felt less consistent with how it's handled in the `ready_to_verify` presenter/view. I think this would be preferable if the presenter used PostOffice as well.